assets/css/stripeStyle.css

```diff
@@ -269,3 +269,17 @@ table tr {
 .summary h2 {
 	color: #304ffe;
 }
+
+.cut-vat {
+	margin-right: 1rem;
+}
+
+.cut-line {
+	width: 59px;
+	height: 2px;
+	background-color: #ffd600;
+	display: inline-block;
+    margin-left: -58px;
```

Comment:
Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5888388)